.github/workflows/test.yml

```diff
@@ -0,0 +1,35 @@
+name: Run tests and analytics
+on:
+  pull_request:
+    branches:
+      - 'master'
+      - 'develop'
+      - 'v*.*.*'
+jobs:
+  test:
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        php_version: [7.4, 8.0]
+        symfony: [4.4, 5]
+        coverage: [none, pcov]
```

Comment:
I'd always install `pcov`. We can decide if we want to run coverage under certain conditions, but might as well be generated for each run.